<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyerInfoForm</class>
 <widget class="QWidget" name="BuyerInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#BuyerInfoForm{
	
	background-color: rgb(255, 245, 247);
}</string>
  </property>
  <widget class="QLabel" name="label_headPic">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>270</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_name">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>名字:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_name">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_1">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>0</y>
     <width>30</width>
     <height>600</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lb_backImage">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>0</y>
     <width>450</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_backImage">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>0</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color:rgb(255,255,255);
}</string>
   </property>
   <property name="text">
    <string>更换封面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_info">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	font-size:20px;
	background-image: url(:/images/loginpushButton.png);
 	background-position: center ;

}</string>
   </property>
   <property name="text">
    <string>修改信息</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_logout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	color:rgb(0,0,0);
}</string>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pawdNew">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>412</y>
     <width>120</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_pawdNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>421</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_pawd">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	font-size:20px;
	background-image: url(:/images/loginpushButton.png);
 	background-position: center ;

}</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_pawdOld">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>363</y>
     <width>120</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>旧密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_pawdOld">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>372</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_sex">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>300</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_sex">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>291</y>
     <width>90</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_id">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>200</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>191</y>
     <width>71</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID   :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
